frontend/src/pages/books/Mobile/BookDetailMobile.tsx

```diff
@@ -0,0 +1,362 @@
+            <div className="flex w-6 justify-content-center">
+              <TextLabel label="Inventory Count" />
+            </div>
+            <div className="flex w-6 justify-content-center">
+              <TextLabel label="Days of Supply" />
+            </div>
+          </div>
+          <div className="flex justify-content-between">
+            <div className="flex w-6 justify-content-center">
+              <p className="p-component p-text-secondary text-900 text-xl text-center my-auto">
+                {stock}
+              </p>
+            </div>
+            <div className="flex w-6 justify-content-center">
+              <p className="p-component p-text-secondary text-900 text-xl text-center my-auto">
+                {daysOfSupply < 0 ? 0 : daysOfSupply}
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-12 justify-content-start p-1">
+          <div className="flex justify-content-between">
+            <div className="flex w-6 justify-content-center">
+              <TextLabel label="Best Buyback Price" />
+            </div>
+            <div className="flex w-6 justify-content-center">
+              <TextLabel label="Retail Price" />
+            </div>
+          </div>
+          <div className="flex justify-content-between">
+            <div className="flex w-6 justify-content-center">
+              <p className="p-component p-text-secondary text-900 text-xl text-center my-auto">
+                {PriceTemplate(bestBuybackPrice)}
+              </p>
+            </div>
+            <div className="flex w-6 justify-content-center">
+              <p className="flex p-component p-text-secondary text-900 text-xl text-center mx-0 my-auto">
+                {PriceTemplate(price)}
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-12 justify-content-start p-1">
+          <div className="flex justify-content-between">
+            <div className="flex w-6 justify-content-center">
+              <TextLabel label="# of Related Books" />
+            </div>
+            <div className="flex w-6 justify-content-center">
+              <TextLabel label="Last Month Sales" />
+            </div>
+          </div>
+          <div className="flex justify-content-between">
+            <div className="flex w-6 justify-content-center">
+              <p className="p-component p-text-secondary text-900 text-xl text-center my-0">
+                {numOfRelatedBooks}
+              </p>
+            </div>
+            <div className="flex w-6 justify-content-center">
+              <p className="p-component p-text-secondary text-900 text-xl text-center my-auto">
+                {lastMonthSales}
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <Divider align="center">
+        <div className="inline-flex align-items-center">
+          <b>Book Related Transactions and Adjustments</b>
```

Comment:
This should be removed, right?

Reply:
The mobile interface requirement said the Book detail requirement had to be implemented which is what I did, but we can have professor elaborate more upon it in class.